common/src/models/domain/account.ts

```diff
@@ -0,0 +1,76 @@
+import { OrganizationData } from '../data/organizationData';
+
+import { KdfType } from '../../enums/kdfType';
+import { StorageKey } from '../../enums/storageKey';
+
+export class Account {
+    userId: string;
+    email: string;
+    information: Map<string, any> = new Map<string, any>();
```

Comment:
correct

Reply:
Could we move it to `activeAccountService`? I really want to avoid anyone ever directly accessing the information.

Reply:
This seems like an over-complication to me. While I agree that accessing `account.Information` directly from something like a return value wouldn't fit the patterns in use elsewhere, there may be a need for it, and I don't see anything particularly _wrong_ with it. Moving `information` to the `activeAccountService` in some fashion won't work, because then we would never maintain in memory information of non-active accounts, which we will certainly need (maybe soon for vault timeouts). Likewise, the only way to ever access `accounts` is to subscribe to them from the `accountsManagementService`, so there still isn't a lot of options for direct control here. With all that in mind, I think we should just leave this as is.